Cython/Utility/Exceptions.c

```diff
@@ -22,17 +22,13 @@ if (likely(__Pyx_init_assertions_enabled() == 0)); else
   #define __pyx_assertions_enabled() (__pyx_assertions_enabled_flag)
 
   static int __Pyx_init_assertions_enabled(void) {
-    PyObject *builtins, *debug, *debug_str;
+    PyObject *builtins, *debug;
     int flag;
-    builtins = PyEval_GetBuiltins();
+    builtins = PyEval_GetBuiltins();  // borrowed
     if (!builtins) goto bad;
-    debug_str = PyUnicode_FromStringAndSize("__debug__", 9);
-    if (!debug_str) goto bad;
-    debug = PyObject_GetItem(builtins, debug_str);
-    Py_DECREF(debug_str);
+    debug = PyDict_GetItem(builtins, PYIDENT("__debug__"));  // borrowed
```

Comment:
I spelled this out intentionally at the time because I didn't want to add the `"__debug__"` string constant to modules (and Python versions) that don't use it. This is one-time initialisation code, not worth keeping a live name constant reference around.

Also, I probably used `PyObject_GetItem()` instead of `PyDict_GetItem()` to avoid the borrowed reference. There's no need for a borrowed reference here either. They are generally risky things and this is Limited API code, so we don't know which implementation it will run with.